rskj-core/src/main/resources/reference.conf

```diff
@@ -309,9 +309,11 @@ rpc {
     providers {
         web {
             cors = "localhost"
+            max_payload_size = 52428800 # 5mb
             http {
                 enabled = true
                 bind_address = localhost
+                max_batch_requests_size = 5
```

Comment:
Just for reviewers to know, why was value `5` selected for the max number of requests?

Reply:
No reasoning behind this, expecting to discuss this in the PR